source/Applications/Advanced/Downsample/Downsample.cpp

```diff
@@ -182,16 +38,14 @@ int main()
         const auto pointCloud = frame.pointCloud();
         std::cout << "Loaded " << pointCloud.size() << " data points" << std::endl;
 
-        auto downsamplingFactor = 4;
-        auto pointCloudDownsampled = downsample(pointCloud, downsamplingFactor);
+        const auto downsamplingFactor = Zivid::PointCloud::Downsampling::by2x2;
```

Comment:
Add the following line before
`std::cout << "Downsampling point cloud" << std::endl;`